Update Kubernetes installation-configuration guide (#1466)
pulumi/registry: themes/default/content/registry/packages/kubernetes/installation-configuration.md

fail to authenticate against the cluster. Set one of these locations to a valid kubeconfig file, if you have not done so
already.

> You can run `kubectl get pods` or a similar command to verify that the cluster is up and configured correctly.
> 
> Once the cluster is accessible, setup is complete and you can follow our [getting started guide]({{< relref "/docs/get-started/kubernetes">}}) or explore some of our [code examples](https://github.com/pulumi/examples#kubernetes).

Reviewer: ```suggestion
Verify the cluster is configured and up by running `kubectl get pods`.

Then follow the [getting started guide]({{< relref "/docs/get-started/kubernetes">}}) or explore some [code examples](https://github.com/pulumi/examples#kubernetes).
```
